hydroflow_lang/src/graph/flat_graph_builder.rs

```diff
@@ -0,0 +1,672 @@
+use std::borrow::Cow;
```

Comment:
Can basically assume this file is the same as what was previously in flat_graph.rs